src/detect-http-raw-header.c

```diff
@@ -186,7 +186,7 @@ static InspectionBuffer *GetData(DetectEngineThreadCtx *det_ctx,
             tx_ud->request_headers_raw : tx_ud->response_headers_raw;
         if (data == NULL)
             return NULL;
-        const uint8_t data_len = ts ?
+        const uint32_t data_len = ts ?
```

Comment:
Yikes, thanks for fixing this.

Reply:
I wonder if there are more of this kind of integer loss of precision, and if there are tools to find them accurately...